lib/generators/authentication/templates/controllers/api/identity/email_verifications_controller.rb.tt

```diff
@@ -24,7 +26,7 @@ class Identity::EmailVerificationsController < ApplicationController
     <%- else -%>
       @token = EmailVerificationToken.find_signed!(params[:sid]); @user = @token.user
     rescue
-      render json: { error: "That email verification link is invalid" }, status: :bad_request
+      render json: { error: "That email verification code is invalid" }, status: :bad_request
```

Comment:
Why code here? I believe it's the verification by email token

Reply:
Code is the word that was already in the controller, so I just changed it here to match - should it say `link` in both places?

Reply:
If it's code_verifiable?, it should a code is invalid... if it's email verification it should be link is invalid.

Reply:
I think that's the confusion. From what we talked about earlier we weren't going to cover tests for the optional configs. Maybe for now it makes the most sense to just change them both to link then?

Reply:
Just pushed code to make that change